node/src/accountant/scanners.rs

```diff
@@ -2248,15 +2738,16 @@ mod tests {
     fn remove_timestamp_and_log_if_timestamp_is_correct() {
         init_test_logging();
         let test_name = "remove_timestamp_and_log_if_timestamp_is_correct";
-        let time_in_past = SystemTime::now().sub(Duration::from_secs(10));
         let logger = Logger::new(test_name);
         let mut subject = ScannerCommon::new(Rc::new(make_custom_payment_thresholds()));
-        subject.initiated_at_opt = Some(time_in_past);
+        let now = SystemTime::now();
```

Comment:
This timestamp stuff would be easier to understand if you picked two hard-coded timestamps (perhaps named `start` and `end`) instead of having one of them be `SystemTime::now()`.